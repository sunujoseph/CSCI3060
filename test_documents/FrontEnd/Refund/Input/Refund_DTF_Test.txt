login 
john123smith 
refund 
Sony PlayStation 4 
john123smith 
ken_adams56 
200 
time 
logout 