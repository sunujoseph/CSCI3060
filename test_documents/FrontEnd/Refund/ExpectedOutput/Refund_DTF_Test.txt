Please log in to the system: 
Please enter your username: 
Enter command: 
Enter Item Name: 
Enter buyer username: 
Enter seller username: 
Enter amount of credit: 
Your request for refund for your Sony PlayStation 4 from ken_adams56 for $200 has been sent...
Your refund for your Sony PlayStation 4 has been processed. 
$200 of credit from ken_adams56's account has been removed and has be deposited into john123smith's account.
Time is: 11:59:48pm
Recording Transaction into Daily Transaction File...
ERROR: Recording of Daily Transaction File Failed. Try again later.
Goodbye!